cmd/workplace/groupSend/main.go

```diff
@@ -0,0 +1,99 @@
+// groupSend is a cli for usage worksplace SendGroup function
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/bornlogic/wiw/workplace"
+)
+
+// if access token isn't passed by flag, use value of env `WORKPLACE_ACCESS_TOKEN`
+var DefaultAccessToken = os.Getenv("WORKPLACE_ACCESS_TOKEN")
+
+// args for function for send to GroupSend
+var accessToken, groupID, formatting, message string
+
+// print verbose mode
+var isVerbose bool
+
+func main() {
+	flag.Parse()
+
+	if err := checkArgs(accessToken, groupID, formatting, message); err != nil {
+		log.Fatal(err)
+	}
+
+	verbose := func(s string) {
+		if isVerbose {
+			log.Println(s)
+		}
+	}
+
+	verbose("do group send")
+	resp, err := workplace.GroupSend(accessToken, groupID, formatting, message)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		log.Fatalf("invalid status returned: %s", resp.Status)
+	}
+
+	verbose("read response")
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalf("read response: %s", err)
+	}
+	defer resp.Body.Close()
+
+	verbose(fmt.Sprintf("response: %s", string(b)))
+}
+
+func checkArgs(accessToken, groupID, formatting, message string) error {
+	if accessToken == "" {
```

Comment:
Coloquei a sugestão pelo navegador, não garanto que funciona, rsrs. MAs eu trataria "nill" e "string apenas com espaços" em todos esses IF

```suggestion
        if accessToken == nil || len(strings.TrimSpace(accessToken)) == 0 {
```

Reply:
o TrimSpace até pode fazer sentido caso passem uma string apenas com espaço, mas o `nil` não tem como acontecer, `strings` em go sempre retornarão strings, só se fosse um ponteiro poderia retornar nil

Reply:
na verdade, olhando melhor, nem o trim faz sentido, internamente o flag ja cuida disso, então se tu não passar a flag ou passar ela vazia tipo "     ", ele vai apitar erro do mesmo jeito, eg:
```
$ go run cmd/workplace/groupSend/main.go --accessToken "    "
flag provided but not defined: -accessToken
```

```
$ go run cmd/workplace/groupSend/main.go
2020/04/03 11:14:18 missing accessToken
```

Reply:
mas muito obrigado pelo review